### ISSUE TITLE
x11-wm/gala meta-background-image.h: No such file or directory

### ISSUE THREAD

**Issue by david2012brooke** [pimvullers/elementary#163]
Received error while compiling x11-wm/gala-0.2.0 and x11-wm/gala-9999 for Gentoo linux.
"meta-background-image.h: No such file or directory"
I have attached the build.log. Please let me know if you need further ifo.
Thanks

[gala-error.txt](https://github.com/pimvullers/elementary/files/644897/gala-error.txt)

**Comment by divoxx** [comment]
@david2012brooke Are you using's dantrell-gnome overlay to install the gnome dependencies with support for OpenRC? 

If so, disable the `deprecated-background` use flag and gala will compile fine. Also, be aware that you need to use `gala-9999` at this point otherwise slingshot will fail to compile.

**Comment by david2012brooke** [comment]
Yes I was using "dantrell-gnome overlay" to overcome the systemd
dependencies.
I will re-try the openrc test again.
On another but a related issue (that I didn't document)...I tried the
systemd route and I believe that I did run into a error with "slingshot" so
I will use gala-9999 recommendation as well.
Thanks
